src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -8883,12 +8806,12 @@ static void Main()
             else
             {
                 comp.VerifyEmitDiagnostics(
-                    // (6,9): error CS8350: This combination of arguments to 'A.MayCaptureRef(__arglist)' is disallowed because it may expose variables referenced by parameter '__arglist' outside of their declaration scope
-                    //         MayCaptureRef(__arglist(ref r)); // error: may expose variables outside of their declaration scope
-                    Diagnostic(ErrorCode.ERR_CallArgMixing, "MayCaptureRef(__arglist(ref r))").WithArguments("A.MayCaptureRef(__arglist)", "__arglist").WithLocation(6, 9),
-                    // (6,37): error CS8168: Cannot return local 'r' by reference because it is not a ref local
-                    //         MayCaptureRef(__arglist(ref r)); // error: may expose variables outside of their declaration scope
-                    Diagnostic(ErrorCode.ERR_RefReturnLocal, "r").WithArguments("r").WithLocation(6, 37));
+                    // (7,9): error CS8350: This combination of arguments to 'A.F2(ref R, ref R)' is disallowed because it may expose variables referenced by parameter 'x2' outside of their declaration scope
```

Comment:
>(7,9): error CS8350: This combination of arguments to 'A.F2(ref R, ref R)' [](http://example.com/codeflow?start=23&length=74)

This does not appear to be error from this test. Consider unskipping the test or restoring the previous errors.